runtime/nvqir/NVQIR.cpp

```diff
@@ -635,7 +635,8 @@ void releasePackedQubitArray(Array *a) {
 /// @param QISFunction
 /// @param
 void invokeWithControlQubits(const std::size_t nControls,
-                             void (*QISFunction)(Array *, Qubit *), ...) {
+                             void (*QISFunction)(Array *, Qubit *, double),
```

Comment:
I found some code in unittests/qir/NVQIRTester.cpp that suggests this signature should be `void (double, Array*, Qubit*)`.

```c++
void __quantum__qis__rx__ctl(double x, Array *ctrls, Qubit *q);
```